Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -136,6 +136,11 @@ if (DEVELOPER_MODE AND DEVELOPER_MODE_FATAL_WARNINGS)
     endif ()
 endif ()
 
+if (DEVELOPER_MODE AND ARM)
+    # Make sure we get good back traces.
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS("-fno-omit-frame-pointer")
+endif ()
```

Comment:
Why only on ARM?

Notably, Fedora also uses `-mno-omit-leaf-frame-pointer` on aarch64 and x86_64, so consider whether you need that too (probably).